babel.config.js

```diff
@@ -109,7 +109,7 @@ module.exports = function (api) {
         "@babel/proposal-object-rest-spread",
         { useBuiltIns: true, loose: true },
       ],
-      compileDynamicImport ? dynamicImportUrlToPath : null,
+      dynamicImportUrlToPath,
       compileDynamicImport ? "@babel/plugin-proposal-dynamic-import" : null,
```

Comment:
Why is this change necessary?

Reply:
Ah I see. `import(url.pathToFileURL())` is meant to be published. I have added a new `test-legacy` BABEL_ENV to `babel.config.js`.